### PR TITLE
:arrow_up: rollup / eslint-plugin-babel

### PR DESCRIPTION
**Pull request by guillaumecrespel**
close #16 
close #17 
close #18 
close #19

### CODE REVIEW

File: misc/rollup.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import path from 'path'
 import fs from 'fs'
 import babel from 'rollup-plugin-babel'
 import commonjs from 'rollup-plugin-commonjs'
-import uglify from 'rollup-plugin-uglify'
+import { uglify } from 'rollup-plugin-uglify'
```
[the remark this answers is not in the document]
Comment (guillaumecrespel) [reply misc/rollup.config.js]: no, What do you have in mind ?

[the remark this answers is not in the document]
Comment (fabienjuif) [reply misc/rollup.config.js]: I can't remember, but I had some issue with k-redux-factory.
This is one of the reason I move to `microbundle`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   lockfile:
     docker:
-      - image: circleci/node:latest
+      - image: circleci/node:9
```
[the remark this answers is not in the document]
Comment (guillaumecrespel) [reply .circleci/config.yml]: yes, but I'm in node V9.
And all stack is with this version. I prefer to fix the version to 9, no ?

[the remark this answers is not in the document]
Comment (fabienjuif) [reply .circleci/config.yml]: I don't think this is a good idea.

I prefer stick to latest versions.
I think in like 1 month, the react-create-app stack would work on node:10, so sticking to `latest` where we can is better for maintenance

File: misc/rollup.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (guillaumecrespel) [reply misc/rollup.config.js]: image

File: .circleci/config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (guillaumecrespel) [reply .circleci/config.yml]: hum .. but that implies that up my node version.
if it's ok for, you, ok or me ;)

[the remark this answers is not in the document]
Comment (fabienjuif) [reply .circleci/config.yml]: `if it's ok for, you, ok or me ;)` ??

[the remark this answers is not in the document]
Comment (fabienjuif) [reply .circleci/config.yml]: And yes upgrade your node, you can use `nvm`

File: tests/__snapshots__/index.spec.js.snap [hunk 3]
```diff
@@ -161,7 +161,7 @@ Object {
     "test": "avec 50% paramètre",
   },
   "number.withUSDFormat": Object {
-    "test": "avec US$ 1,234.00 paramètre",
+    "test": "avec $1,234.00 paramètre",
```
Comment (fabienjuif) [inline tests/__snapshots__/index.spec.js.snap]: nope, run test with node 10

Comment (fabienjuif) [reply tests/__snapshots__/index.spec.js.snap]: ok tests passes. I don't understand anymore.

`$` is node 10 ?
`US$` is node 9 ?

Comment (guillaumecrespel) [reply tests/__snapshots__/index.spec.js.snap]: I don't know :/ On local , with node 10, I've the same issue that previous, ...

Comment (fabienjuif) [reply tests/__snapshots__/index.spec.js.snap]:
```
➜  k-intl git:(upVersion) nvm use 10
Now using node v10.4.0 (npm v6.1.0)
➜  k-intl git:(upVersion) yarn test
yarn run v1.7.0
$ jest
 PASS  tests/index.spec.js
 PASS  src/utils/utils.spec.js

Test Suites: 2 passed, 2 total
Tests:       5 passed, 5 total
Snapshots:   6 passed, 6 total
Time:        1.05s
Ran all test suites.
✨  Done in 1.47s.
```

Comment (fabienjuif) [reply tests/__snapshots__/index.spec.js.snap]:
```
➜  k-intl git:(upVersion) nvm use 9
Now using node v9.8.0 (npm v5.6.0)
➜  k-intl git:(upVersion) yarn test
yarn run v1.7.0
$ jest
 FAIL  tests/index.spec.js
  ● src/format › should format message with FR Locale

    expect(value).toMatchSnapshot()

    Received value does not match stored snapshot 1.

    - Snapshot
    + Received

    @@ -16,11 +16,11 @@
        },
        "number.withPercentParam": Object {
          "test": "avec 50% paramètre",
        },
        "number.withUSDFormat": Object {
    -     "test": "avec $1,234.00 paramètre",
    +     "test": "avec US$ 1,234.00 paramètre",
        },
        "number.withoutParam": Object {
          "test": "aucun paramètre",
        },
        "plural.simple.noparam": Object {

      49 | const tester = (lang, locale) =>
      50 |   it(`should format message with ${lang} Locale`, () => {
    > 51 |     expect(keyValueResources(lang, format(lang, locale, customFormats))).toMatchSnapshot()
      52 |     expect(keyValueResources(lang, format(lang, undefined))).toMatchSnapshot()
      53 |   })
      54 |

      at Object.<anonymous> (tests/index.spec.js:51:74)

 › 1 snapshot test failed.
 PASS  src/utils/utils.spec.js

Snapshot Summary
 › 1 snapshot test failed in 1 test suite. Inspect your code changes or run `yarn test -u` to update them.

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 4 passed, 5 total
Snapshots:   1 failed, 5 passed, 6 total
Time:        1.237s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
➜  k-intl git:(upVersion)
```

**Comment by guillaumecrespel** [comment]
``
`
➜  k-intl git:(upVersion) nvm use 10
Now using node v10.4.0 (npm v6.1.0)
➜  k-intl git:(upVersion) yarn test
yarn run v1.5.1
(node:9742) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
$ jest
 FAIL  tests/index.spec.js
  ● src/format › should format message with FR Locale

    expect(value).toMatchSnapshot()

    Received value does not match stored snapshot 1.

    - Snapshot
    + Received

    @@ -16,11 +16,11 @@
        },
        "number.withPercentParam": Object {
          "test": "avec 50% paramètre",
        },
        "number.withUSDFormat": Object {
    -     "test": "avec $1,234.00 paramètre",
    +     "test": "avec US$ 1,234.00 paramètre",
        },
        "number.withoutParam": Object {
          "test": "aucun paramètre",
        },
        "plural.simple.noparam": Object {

      49 | const tester = (lang, locale) =>
      50 |   it(`should format message with ${lang} Locale`, () => {
    > 51 |     expect(keyValueResources(lang, format(lang, locale, customFormats))).toMatchSnapshot()
      52 |     expect(keyValueResources(lang, format(lang, undefined))).toMatchSnapshot()
      53 |   })
      54 |

      at Object.<anonymous> (tests/index.spec.js:51:74)

 › 1 snapshot test failed.
 PASS  src/utils/utils.spec.js

Snapshot Summary
 › 1 snapshot test failed in 1 test suite. Inspect your code changes or run `yarn test -u` to update them.

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 failed, 4 passed, 5 total
Snapshots:   1 failed, 5 passed, 6 total

```

**Comment by fabienjuif** [comment]
```
➜  k-intl git:(upVersion) env | grep LC
LC_CTYPE=en_GB.UTF-8
➜  k-intl git:(upVersion) env | grep LANG
LANG=en_GB.UTF-8
➜  k-intl git:(upVersion) yarn --version
1.7.0
```

**Pull request merged**